core/storage/classifier/gae_models_test.py

```diff
@@ -90,3 +90,25 @@ def test_create_and_get_new_training_job_runs_successfully(self):
                          feconf.TRAINING_JOB_STATUS_NEW)
         self.assertEqual(training_job.training_data,
                          [{'answer_group_index': 1, 'answers': ['a1', 'a2']}])
+
+
+class ClassifierExplorationMappingModelUnitTests(test_utils.GenericTestBase):
+    """Tests for the ClassifierExplorationMappingModel class."""
+
+    def setUp(self):
+        super(ClassifierExplorationMappingModelUnitTests, self).setUp()
+        classifier_models.ClassifierExplorationMappingModel.create(
+            'exp_id1', 1, 'state_name1', 'classifier_id1')
```

Comment:
These and the other tests need to include state names that have weird unicode characters. It's possible that you might need to do some encoding to ascii before saving to datastore in order to avoid strange chars in the IDs.